test/javascript/components/student/iteration-page/EmptyIterations.test.js

```diff
@@ -50,6 +51,7 @@ test('disables buttons when loading', async () => {
       />
     </TestQueryCache>
   )
+  await awaitPopper()
```

Comment:
This test gave me a warning when run with `yarn test-debug`, so I've fixed it using this.